lib/config/config.js

```diff
@@ -29,6 +32,35 @@ var isInitialized = false;
 loggingService.addWriter('terminal', ['>=notice'], {});
 
 /**
+ * ConfigurationError class
+ */
+class ConfigurationError extends Error {
+	constructor(message, filename, error) {
+		super(message);
+
+		this.name = 'ConfigurationError';
```

Comment:
Does extending `Error` not do this for you?

Reply:
```shell
> class H extends Error{}
[Function: H]
> (new H('asd')).name
'Error'
```

Reply:
I could've tested that. Sorry for the noise.